### PR TITLE
[MM-47805] Remove CRT feature flag

### PR DESCRIPTION
**Pull request by vish9812**
#### Summary

Remove the CRT feature flag given it’s out in GA and stable.

#### Ticket Link

JIRA - https://mattermost.atlassian.net/browse/MM-47805

#### Related Pull Requests

- Server - (https://github.com/mattermost/mattermost-server/pull/21788)
- Mobile - (https://github.com/mattermost/mattermost-mobile/pull/6848)

#### Release Note

```release-note
Removed the CRT feature flag.
```

### CODE REVIEW

File: components/permalink_view/permalink_view.test.tsx [hunk 1]
```diff
@@ -304,7 +306,6 @@ describe('components/PermalinkView', () => {
                         ...initialState.entities,
                         general: {
                             config: {
-                                FeatureFlagCollapsedThreads: 'true',
```
Comment (Willyfrog) [inline components/permalink_view/permalink_view.test.tsx:307]: this looks like the only change, we should avoid changing the rest of the file unless needed.

**Review by Willyfrog — APPROVED** [review]

File: components/permalink_view/permalink_view.test.tsx (hunk 1 above)
Comment (vish9812) [reply components/permalink_view/permalink_view.test.tsx:307]: These are actually auto lint changes done by `make fix-style` command.

**Review by koox00 — APPROVED** [review]

**Review by koox00 — CHANGES_REQUESTED** [review]
We need to remove the feature flag checks in admin console, `admin-definition.tsx`, as well

e.g: https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/admin_definition.jsx#L2787

**Comment by vish9812** [comment]
@koox00 Updated it, thanks.

**Review by koox00 — APPROVED** [review]

**Review by jgilliam17 — APPROVED** [review]
Tested on related server PR. LGTM

**Pull request merged**